spec/models/task.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+RSpec.describe 'タスク管理機能', type: :model do
+  it 'titleが空ならバリデーションが通らない' do
+    task = Task.new(name: '', detail: '失敗テスト')
+    #taskが妥当（valid）でないことを期待する
+    expect(task).not_to be_valid
+  end
+  it 'detailが空ならバリデーションが通らない' do
+    task = Task.new(name: '失敗テスト', detail: '')
```

Comment:
Factoryを使うと下記のように記載できることも覚えておいてください。
```suggestion
    task = FactoryBot.build(:task, name: '失敗テスト', detail: '')
```